### ISSUE TITLE
Criar workflows para modulos

### ISSUE THREAD

**Issue by arthurTemporim** [lappis-unb/rasa-ptbr-boilerplate#157]
# Problema
Atualmente existe apenas o workflow que faz a build do bot. Porém o boilerplate possui outros módulos e efeutar o teste destes módulos é necessário.

# Sugestão de solução
Configurar um arquivo de workflow buildando e testando cada um dos módulos do boilerplate.
Caso necessário criar uma "subissue" pra cada módulo e assim ficar rastreando melhor a evolução desta contribuição.

[workflows](https://github.com/lappis-unb/rasa-ptbr-boilerplate/actions)

## Tarefas
- [ ] Configurar workflow de analytics
- [ ] Configurar workflow de elasticsearch
- [ ] Configurar workflow de notebooks
- [ ] Configurar workflow de rabbitmq
- [ ] Configurar workflow de rocketchat
- [ ] Configurar workflow de webchat

> Usem o espaço da issue para discutir sobre as soluções

**Comment by RochaCarla** [comment]
Essas tarefas todas são a mesma issue ou cada workflow seria uma issue distinta? pode falar sobre uma posssivel solução..

**Comment by BrunaNayara** [comment]
Acho melhor abrirmos uma issue pra cada workflow. Acho que tem algumas que podem ser simples, mas tem algumas que podem dar um trabalhinho a mais

**Comment by arthurTemporim** [comment]
Concordo com a Bruna, vai ser melhor pra acompanhar cada uma separada, eu só coloquei na mesma issue pra não criar todas de uma vez, pode ser que algum suma ou mais apareçam.

Então pode ter essa issue como um "ÉPICO" e linkar cada um dos workflows que forem feitos aqui

**Comment by RochaCarla** [comment]
Massa! vou abrir aqui
